### DISCUSSION TITLE
Repeat in a circle

### DISCUSSION THREAD

**Discussion by audiovaccine** [lumetrium/obsidian-teleprompter#4] [Ideas]
Hello and thank you very much for the plug-in, it's great!
Could you please add an option to start from the beginning when the file is over?

**Comment by lumetrium** [comment]
Hey, thanks for the suggestion! Do you have a specific use-case in mind? It shouldn't be too difficult to add, but I'm looking to understand the situations in which it would be useful.

**Reply by trentdigital** [reply]
Thank you for your work on this plugin - it's been an absolute game changer for me and could benefit many others! I wanted to provide a few scenarios where a "restart from beginning" loop would be incredibly beneficial:

Personal Productivity & ADHD Support
I use the teleprompter on an old phone placed below my monitor to auto-scroll through reminders, tasks, and mindset shifts. I have obsidian sync so I can update the page from my main terminal and it updates on the device. I pin the window and expand full screen and with dedicated power it just runs. This is actually a clinically-recognized approach to help people with ADHD bridge the gap between intention and action by providing external scaffolding for executive function and really helps with new habit formation. Looping would eliminate manual restarts and make this a true persistent aid. 

Information Displays, Alerts / News Ticker & Announcements

When it loops automatically it effectively changes this into a ticker ( I've tried a lot of news/rss/update tickers) 

Office environments: Meeting schedules, deadlines, company updates, announcements as people come and go, 
Schools: Test times, schedule changes, important announcements, school concerts itinerary
Sports events: Tournament schedules, game changes, venue information. Anytime there's a change in a field, court, this turns this into an alert ticker. 
Events (weddings, conferences): Itineraries, location details, timing updates where you need the same information to go to a group of people that are coming in at different times. 
Financial environments: Market updates, key metrics, team goals


Learning & Reference Material
Cycling through information repeatedly supports retention when learning new material or when reference information needs to be continuously visible.

A simple toggle option to "Loop content" would dramatically expand the utility of this plugin while keeping its elegant simplicity.
Thanks for considering this addition!

**Reply by lumetrium** [reply]
Thanks! I love these, especially the announcements use-case. I'm going to implement this feature, although I'm not quite sure when. Pretty busy right now, but I'll try to find some time.
